src/core/border_router/routing_manager.cpp

```diff
@@ -149,6 +149,7 @@ Error RoutingManager::GetFavoredOmrPrefix(Ip6::Prefix &aPrefix, RoutePreference
     Error error = kErrorNone;
 
     VerifyOrExit(IsInitialized(), error = kErrorInvalidState);
+    VerifyOrExit(Get<Mle::Mle>().IsAttached(), error = kErrorInvalidState);
```

Comment:
We should check against the RoutingManager status. For example, this is only available when `mIsRunning` is true?

Reply:
Good point!